packages/lit-dev-content/samples/tutorials/svg-templates/tutorial.json

```diff
@@ -0,0 +1,35 @@
+{
+  "header": "SVG Templates",
```

Comment:
This is in "Build" section but the title or card description don't really seem to convey that it's a "build" but rather learn